.github/workflows/humble-abi-compatibility.yml

```diff
@@ -8,7 +8,7 @@ on:
 concurrency:
   # cancel previous runs of the same workflow, except for pushes on humble branch
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ github.ref != 'humble' }}
+  cancel-in-progress: ${{ !startsWith(github.ref, '/refs/heads') }}
```

Comment:
This lets any branch head continue. Is that what you intend?

It's probably fine since this repo seems to keep a small set of branches.

Reply:
what I want here is that old workflows after pushes to PRs get cancelled, but wfs after pushes to the branches (master, humble..) aren't. so that we save CI time in PRs but have tests for every commit on the head branches.